### DISCUSSION TITLE
Carlinkit CCPA

### DISCUSSION THREAD

**Discussion by paulharman** [tesla-android/issue-tracker#35] [Q&A] [answered]
Hi, is there any reason to get or not to get the new version of CCPA with the microphone?
![image](https://user-images.githubusercontent.com/3407038/171027437-4837aadd-dfe3-4ad4-97c3-49d4a587467d.png)

**Comment by mikegapinski** [comment, marked as answer]
It will work just fine, I even suggested CCPA in the hardware requirements if someone is looking for wireless android auto

**Reply by paulharman** [reply]
Thanks for the reply....I know CCPA is working fine, but they have a new version of CCPA with a microphone - I don't know if this is used?

**Reply by mikegapinski** [reply]
It will work, you don't need the version with the microphone since this functionality is covered by your Tesla using Bluetooth

**Reply by paulharman** [reply]
perfect., thank you
